victor_description/urdf/victor.urdf.xacro

```diff
@@ -42,30 +44,35 @@
     <link name="$(arg robot_name)_right_arm_world_frame_kuka">
     </link>
 
-
     <!-- arm to gripper adapter -->
     <xacro:include filename="$(find victor_description)/urdf/iiwa7_to_robotiq_adapter.xacro"/>
     <xacro:iia7_to_robotiq_adapter parent="$(arg robot_name)_left_arm_flange" prefix="$(arg robot_name)_left">
-        <origin xyz="0 0 0" rpy="0 0 0"/>
+        <origin xyz="0 0 -0.01" rpy="0 0 0"/>
```

Comment:
Is the `-0.01` just a guess? We might have to fine-tune on real victor

Reply:
I have no idea what this is or where it came from lmao